flask_security/forms.py

```diff
@@ -44,8 +44,10 @@
 
 class ValidatorMixin(object):
     def __call__(self, form, field):
+        # This is a fix that avoids the API being changed.
         if self.message and self.message.isupper():
-            self.message = get_message(self.message)[0]
+            self.message_key = self.message
+        self.message = get_message(self.message_key)[0]
```

Comment:
What if the `self.message_key` is not set?

Can we change the Validator mixin to something around this lines?
```python
class ValidatorMixin(object):
   
    @property
    def message(self):
        if self._templated:
            return get_message(self._message)[0]
        return self._message

    @message.setter
    def message(self, value):
        self._templated = value and value.isupper()
        self._message = value
```

Reply:
Why did I miss that? :)

I'm ok with properties.
Do you want me to put it into this Pull Request?

Reply:
@kantorii yes, you can update this PR.

Reply:
@jirikuncar Thanks for the wait. The suggested change is on the branch now.